mp/src/game/shared/momentum/mom_system_gamemode.cpp

```diff
@@ -121,6 +128,10 @@ void CGameModeSystem::LevelInitPostEntity()
     m_pCurrentGameMode->SetGameModeVars();
     PrintGameModeVars();
 #endif
+
+#ifdef CLIENT_DLL
```

Comment:
Because a file is compiled into either server (GAME_DLL) or client (CLIENT_DLL), you're safe to do an `#else` here instead of a separate `#ifdef` check. E.g.

```
#ifdef GAME_DLL
   . . . // server code
#else
 . . . // Client code, implicitly CLIENT_DLL
#endif
```

Reply:
Alright I'll do this

Reply:
It is done